CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # go-ipfs changelog
 
+## v0.14.0 TBD
+
+### Overview
+
+- Delegated routing using [Reframe protocol](https://github.com/ipfs/specs/blob/main/REFRAME.md)
```

Comment:
We're going to want to expand this to explain why it's useful and point to how someone uses it.  I think we'll want a code snippet for configuring to hit storetheindex.

Reply:
The snippet for adding `https://cid.contact/reframe` via CLI:

```
ipfs config Routing.Routers.DevCidContact --json '{
  "Type": "reframe",
  "Parameters": {
    "address": "https://cid.contact/reframe"
  }
}'
```

As for text, there is some content  on https://cid.contact/ and https://docs.cid.contact that could be copied or linked to.